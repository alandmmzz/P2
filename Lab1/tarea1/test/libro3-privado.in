crearFecha 16/07/1951
crearLibro 316769177 "El guardián en el centeno" "J.D." "Salinger" "Holden narra sus días tras ser expulsado del colegio, en busca de sentido" 1

imprimirISBNLibro
imprimirTituloLibro
imprimirNombreAutorLibro 
imprimirApellidoAutorLibro 
imprimirDescripcionLibro 
imprimirFechaEdicionLibro 
imprimirIdGeneroLibro 
liberarLibro

crearFecha 01/05/1992
crearLibro 140003446 "Jazz" "Toni" "Morrison" "Una historia de amor y crimen en el Harlem de los años 20" 3

imprimirISBNLibro
imprimirTituloLibro
imprimirNombreAutorLibro 
imprimirApellidoAutorLibro 
imprimirDescripcionLibro 
imprimirFechaEdicionLibro 
imprimirIdGeneroLibro 
liberarLibro

crearFecha 01/01/2008
crearLibro 880619315 "La soledad de los números primos" "Paolo" "Giordano" "Dos almas heridas incapaces de coincidir en el amor" 1

imprimirISBNLibro
imprimirTituloLibro
imprimirNombreAutorLibro 
imprimirApellidoAutorLibro 
imprimirDescripcionLibro 
imprimirFechaEdicionLibro 
imprimirIdGeneroLibro 
liberarLibro

Fin
