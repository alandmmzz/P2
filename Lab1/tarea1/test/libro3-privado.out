1>2>3>El isbn del libro es: 316769177
4>El título del libro es: El guardián en el centeno
5>El nombre del autor del libro es: J.D.
6>El apellido del autor del libro es: Salinger
7>La descripción del libro es: Holden narra sus días tras ser expulsado del colegio, en busca de sentido
8>La fecha de edición del libro es: 16/7/1951
9>El id del género del libro es: 1
10>Función 'liberarTLibro' ejecutada exitosamente.
11>12>13>El isbn del libro es: 140003446
14>El título del libro es: Jazz
15>El nombre del autor del libro es: Toni
16>El apellido del autor del libro es: Morrison
17>La descripción del libro es: Una historia de amor y crimen en el Harlem de los años 20
18>La fecha de edición del libro es: 1/5/1992
19>El id del género del libro es: 3
20>Función 'liberarTLibro' ejecutada exitosamente.
21>22>23>El isbn del libro es: 880619315
24>El título del libro es: La soledad de los números primos
25>El nombre del autor del libro es: Paolo
26>El apellido del autor del libro es: Giordano
27>La descripción del libro es: Dos almas heridas incapaces de coincidir en el amor
28>La fecha de edición del libro es: 1/1/2008
29>El id del género del libro es: 1
30>Función 'liberarTLibro' ejecutada exitosamente.
31>Fin.
